### DISCUSSION TITLE
Shortest path notebook

### DISCUSSION THREAD

**Discussion by NikitaSharma1** [networkx/networkx#5479] [Ideas]
I am trying to create an explanatory notebook on shortest paths. Should I explain dijkstra, floyd warshall and bellman ford first and then explain the shortest_path function or start with shortest_path and then include other algos in between?

**Comment by MridulS** [comment]
I would start with just 1 or 2 shortest path algorithms, as otherwise the notebook may get too long :)
To get an overview of expectations of this notebook please do look at the current one available on the nx-guides website, like https://networkx.org/nx-guides/content/algorithms/flow/dinitz_alg.html

**Reply by NikitaSharma1** [reply]
I created a notebook on "shortest path using Dijkstra's algorithm" [pull request](https://github.com/networkx/nx-guides/pull/65) . Can you please have a look and give feedback?

**Comment by NikitaSharma1** [comment]
@MridulS Should I start working on the Bellman ford notebook now or select some other algorithm? Also, can you please tell me whether I should include bellman ford in the same Dijkstra's notebook or create a new one?
